### DISCUSSION TITLE
Config: Different rules & schedule for a specific subpath of the codebase

### DISCUSSION THREAD

**Discussion by aaomidi** [renovatebot/renovate#31615] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

None

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Not self hosting

### Please tell us more about your question or problem

My configuration currently looks like this:

```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:best-practices"
  ],
  "assigneesFromCodeOwners": true,
  "postUpdateOptions": [
    "gomodTidy",
    "helmUpdateSubChartArchives"
  ],
  "labels": [
    "dependencies"
  ],
  "packageRules": [
    {
      "matchFileNames": [
        "webui/"
      ],
      "groupName": "web-dependencies",
      "extends": [
        "schedule:weekly"
      ],
      "addLabels": [
        "webui"
      ]
    },
    {
      "matchSourceUrls": [
        "https://github.com/pulumi/*"
      ],
      "groupName": "Pulumi Packages"
    }
  ]
}

```

My question is specifically with the first package rule.

My end goal here is that any renovate changes under the `webui/**` path in the codebase gets:
- Updated only once a week
- Does not pin dependencies
- Updates in separate PRs, never mixing with PRs for the rest of the codebase

How can I accomplish this?

### Logs (if relevant)

I don't think there's relevant logs for this?

**Comment by rarkins** [comment]
Your matchFileNames is incorrect - it's not an exact match or a glob pattern. See the examples in https://docs.renovatebot.com/configuration-options/#matchfilenames for help
